### DISCUSSION TITLE
Aggregated view/amount for all asset accounts

### DISCUSSION THREAD

**Discussion by timaschew** [firefly-iii/firefly-iii#6378] [Firefly III] [answered]
Is it possible to show a history for the sum of all my assets accounts.

I would like to have a chart with a line which aggreates all my assets.

Why?
If I do a transfer transaction from my checking account to my cash account then the first account will shrink and the second would grow but I would stlil have the exact same amount of money. And I would like to visualize exactly this in firefly.

I could imagine this could be something like a virtual account which allows to select which asset accounts are aggregated.
On the chart it could be visualized a bit different, for instance as a dashed line.

**Comment by JC5** [comment, marked as answer]
The net worth is a part of the default report. Is that what you're looking for, perhaps?

**Reply by timaschew** [reply]
Yes, indeed!

I did not realize that the report is not avilable via the reports page, but only when I click on the dashboard on **net worth**. Or did I miss something on the reports page?

**Reply by JC5** [reply]
No it's part of the default report.

**Reply by timaschew** [reply]
If I go to the report page and choose `Default financial report`, select `Default asset accounts` and submit, then I see a page with reports which contains lot of things but **not** the **net worth**.

What I see is:
- account balances
- income vs. expenses
-  revenue/income
-  expenses
-  budgets
-  categories
-  budget (split by account)
-  bills

Is it expected to not be able to select the net worth report via the `/reports` page?

**Reply by JC5** [reply]
Oh my bad, make sure the range of the report is more than two months.

**Reply by timaschew** [reply]
Oh, yes that helped, thanks!
